### PR TITLE
Fix Issue #19 (add array extension)

### PR DESCRIPTION
**Pull request by hajgato**

### CODE REVIEW

File: files/qsub.pl [hunk 1]
```diff
@@ -670,6 +671,23 @@ sub parse_script
         };
     };
 
+    # add array extensions if it is an array job
+    if ($orig_argsh{t} || $set{t}) {
+        my $arrayext = '-%a';
+        for (my $element=0; $element < scalar(@cmd); $element++) {
```
Comment (stdweird) [inline files/qsub.pl]: this is perl, not c:
```perl
for my $i (0 .. $#cmd) {
}
```

Comment (hajgato) [reply files/qsub.pl]: done

File: files/qsub.pl [hunk 2]
```diff
@@ -697,6 +698,23 @@ sub parse_script
...
+    if ($orig_argsh{t} || $set{t}) {
+        my $arrayext = '-%a';
+        for my $element (0 .. $#cmd) {
+            foreach my $dir (qw(e o)) {
+                if ($cmd[$element] eq "-$dir" && $cmd[$element+1] !~ m{%a}) {
+                    $cmd[$element+1] .= $arrayext;
+                }
+            }
+        }
+        for (my $element=0; $element < scalar(@newtxt); $element++) {
```
Comment (stdweird) [inline files/qsub.pl]: and this one too

Comment (hajgato) [reply files/qsub.pl]: :D

Comment (hajgato) [reply files/qsub.pl]: done

**Pull request merged**